WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -19,10 +18,13 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets;
 	Searchable = false)]
 public partial class WalletSettingsViewModel : RoutableViewModel
 {
+	private readonly IWalletModel _wallet;
 	[AutoNotify] private bool _preferPsbtWorkflow;
+	[AutoNotify] private string _walletName;
 
 	private WalletSettingsViewModel(IWalletModel wallet)
 	{
+		_wallet = wallet;
 		Title = $"{wallet.Name} - Wallet Settings";
```

Comment:
TBH, it should be just `Wallet Setings` from now

Reply:
Yes, makes sense. Done.